src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -0,0 +1,312 @@
+
+    const signers = ref<any[]>([]);
+    const minNumOfSigners = computed(
+      () => signers.value.length < MULTISIG_VAULT_MIN_NUM_OF_SIGNER
+        ? MULTISIG_VAULT_MIN_NUM_OF_SIGNER
+        : signers.value.length,
+    );
+    const isValidSigners = computed(
+      () => !signers.value.filter((signer) => !validateHash(signer.address).valid).length,
+    );
+    const canCreateMultisig = computed(
+      () => signers.value.length >= MULTISIG_VAULT_MIN_NUM_OF_SIGNER && isValidSigners.value,
+    );
+
+    async function onModalOpen() {
+      await nextTick();
+      // add current active account to signers.
+      signers.value.push({
+        address: account.value.address,
+        isCurrentAccount: true,
+      });
+    }
+
+    function addNewSigner() {
+      signers.value.push({
+        address: '',
+      });
+    }
+
+    /**
+     * Scans a QR code and add a signer address
+     * @param {number} signerIndex - The index of the signer to update the address of
+     */
+    async function openScanQrModal(signerIndex: number) {
+      const scanResult = await root.$store.dispatch('modals/open', {
+        name: MODAL_READ_QR_CODE,
+        title: root.$t('pages.send.scanAddress'),
+        icon: 'critical',
+      });
+
+      if (!scanResult) return;
+
+      const { valid, isName } = validateHash(scanResult);
+
+      // check if the address is valid and it's not a name
+      if (!(valid && !isName)) {
+        root.$store.dispatch('modals/open', {
+          name: MODAL_DEFAULT,
+          title: root.$t('modals.invalid-qr-code.msg'),
+          icon: 'critical',
+        });
+        return;
+      }
+
+      // Check if signer address already added
+      if (signers.value.find((signer) => signer.address === scanResult)) {
+        root.$store.dispatch('modals/open', {
+          name: MODAL_DEFAULT,
+          title: root.$t('modals.createMultisigAccount.error-duplicating-signer'),
+          icon: 'critical',
+        });
+        return;
+      }
+
+      // Update the signer address
+      signers.value[signerIndex].address = scanResult;
+    }
+
+    function getSignerLabel(index: number) {
+      return `${root.$t('modals.createMultisigAccount.signer')} ${index + 1}`;
+    }
+
+    /**
+     * TODO
+     */
+    async function createMultisigVault() {
+      root.$store.dispatch('modals/open', {
+        name: MODAL_DEFAULT,
+        title: 'TODO: implement multisig vault creation process.',
+        icon: 'info',
+      });
+    }
+
+    return {
+      signers,
+      minNumOfSigners,
+      isValidSigners,
+      canCreateMultisig,
+      onModalOpen,
+      createMultisigVault,
+      openScanQrModal,
+      addNewSigner,
+      getSignerLabel,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/typography';
+@use '../../../styles/mixins';
+
+.multisig-vault-create {
+  padding-top: env(safe-area-inset-top);
+
+  .content {
+    padding: 0 24px;
```

Comment:
Modal body has the padding of 24px by default. There is no need to set this here.